src/logger.service.ts

```diff
@@ -133,6 +135,10 @@ export class NGXLogger {
 
     const color = NGXLoggerUtils.getColor(level);
 
-    console.log(`%c${metaString}`, `color:${color}`, message, ...(additional || []));
+    if (customError) {
```

Comment:
This shouldn't change no matter if it was a custom error or a string. The logged string should be the same. This needs to be refactored to not use an ```if / else```